app/controllers/verifications_controller.rb

```diff
@@ -7,6 +7,10 @@ def create
     statement = Statement.find_by!(transaction_id: params.fetch(:id))
     statement.verifications.create!(verification_params)
 
+    # If there was a correction to the name, save that on the
+    # statement so it'll be used for reconciliation and actioning:
+    statement.update_attributes(person_name: params[:new_name]) if params[:new_name]
```

Comment:
Personally I would’ve put this as a callback on the model over in the controller but this does what we need and works

Reply:
@gbp Could you explain in a bit more detail what you mean by that? (I could see how you could do it with a callback on the Verification model, but not sure what you mean by putting it in the controller.)

Reply:
No sure I said anything about putting it in the controller. Maybe I wasn't clear, sorry. I'm advocating keeping the controller action slimmer so its doing only one thing

Reply:
Oh, I see - I thought by "over in the controller" you meant "over there, in the controller" but you meant "rather than in the controller"! :smile: